test/parallel/test-net-localerror.js

```diff
@@ -20,25 +20,22 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
-const assert = require('assert');
+const common = require('../common');
 const net = require('net');
 
-// Using port 0 as localPort / localAddress is already invalid.
+const connect = (opts, code, type) => common.expectsError(
+  () => net.connect(opts),
+  { code, type }
+);
```

Comment:
this doesn't quite match our typical code format .... it should be something like

```js
const connect = (opts, code, type) => {
  common.expectsError(...);
};
```